src/components/link/daikin-link.ts

```diff
@@ -0,0 +1,151 @@
+import { cva } from "class-variance-authority";
+import { LitElement, css, html, unsafeCSS } from "lit";
+import { customElement, property } from "lit/decorators.js";
+import { ifDefined } from "lit/directives/if-defined.js";
+import tailwindStyles from "../../tailwind.css?inline";
+
+const cvaLink = cva(
+  [
+    "flex",
+    "justify-center",
+    "items-center",
+    "gap-0.5",
+    "w-fit",
+    "h-fit",
+    "font-daikinSerif",
+    "rounded-[1px]",
+    "relative",
+
+    "focus-visible:outline-none",
+
+    "focus-visible:before:block",
+    "focus-visible:before:w-[calc(100%+2px)]",
+    "focus-visible:before:h-[calc(100%+2px)]",
+    "focus-visible:before:rounded-[1px]",
+    "focus-visible:before:absolute",
+    "focus-visible:before:outline",
+    "focus-visible:before:outline-system-state-focus",
+    "focus-visible:before:outline-2",
+  ],
+  {
+    variants: {
+      disabled: {
+        false: [
+          "text-system-state-link-active",
+          "hover:text-system-state-link-hover",
+          "active:text-system-state-link-press",
+          "[&>span::after]:bg-system-state-link-active",
+          "[&:hover>span::after]:bg-system-state-link-hover",
+          "[&:active>span::after]:bg-system-state-link-press",
+        ],
+        true: [
+          "text-system-state-disabled",
+          "[&>span::after]:bg-system-state-disabled",
+        ],
+      },
+    },
+  }
+);
+
+/**
+ * The button component is a versatile UI element that triggers actions or submits forms when clicked.
+ * It functions similarly to the HTML `<button>` tag, allowing users to initiate various operations such as submitting data, opening dialogs, or navigating to different sections of an application.
+ *
+ * @slot - A slot for the button content.
+ *
+ * @example
+ *
+ * ```js
+ * import "@daikin-oss/design-system-web-components/components/link/index.js";
+ * ```
+ *
+ * ```html
+ * <daikin-link href="#">
```

Comment:
nit. I don't think `href="#"` be a good practice in these days.

```suggestion
 * <daikin-link href="https://www.example.com">
```